docs/source/models/causes/opioid_use_disorder/index.rst

```diff
@@ -0,0 +1,821 @@
+     - Elevated EMR for untreated OUD, accounting for increased overdose and other mortality risks
+   * - T
+     - Excess mortality rate (EMR)
+     - 0 or reduced value
+     - MOUD substantially reduces mortality risk [Sordo-2017]_ [Santo-2021]_; EMR in the treatment state is assumed to be substantially reduced relative to untreated OUD. In some base-case scenarios we set it to 0 to represent the protective effect of treatment, with sensitivity analyses using non-zero EMR values
+   * - S
+     - Disability weight
+     - 0
+     - No disability burden in susceptible state
+   * - C
+     - Disability weight
+     - :math:`\frac{1}{\text{prevalence}_{\text{c562}}} \times \sum\limits_{s \in \text{sequelae}} \text{disability_weight}_s \times \text{prevalence}_s`
+     - Weighted average of symptomatic and asymptomatic OUD disability weights
+   * - T
+     - Disability weight
+     - 0 or reduced value
+     - MOUD substantially reduces disability [Wakeman-2020]_; disability weight in the treatment state is assumed to be substantially reduced relative to untreated OUD. In some base-case scenarios we set it to 0 to represent an optimistic upper bound on treatment effectiveness, with sensitivity analyses using non-zero disability weights
+
+Transition Data
+~~~~~~~~~~~~~~~
+
+.. list-table:: Transition Data
+   :widths: 8 8 8 25 35
+   :header-rows: 1
+
+   * - Transition
+     - Source
+     - Sink
+     - Value
+     - Notes
+   * - i
+     - S
+     - C
+     - Derived from DisMod-AT/NumPyro model
+     - Incidence rate of OUD. Estimated using NumPyro implementation of DisMod-AT-like model to ensure internal consistency with prevalence, treatment coverage, and other epidemiological parameters.
+   * - r
+     - C
+     - S
+     - Derived from DisMod-AT/NumPyro model
+     - Natural remission rate for untreated OUD. Not directly available from GBD; estimated using DisMod-AT/NumPyro model to ensure consistency with observed prevalence and treatment patterns.
+   * - ti
+     - C
+     - T
+     - Derived from DisMod-AT/NumPyro model and treatment access data
+     - Treatment initiation rate. Estimated based on treatment coverage ratios and population-level treatment access data. May vary by setting (community, jail, etc.) and be modified by intervention scenarios.
+   * - td
+     - T
+     - C
+     - Derived from DisMod-AT/NumPyro model and treatment retention studies
+     - Treatment discontinuation rate. Estimated from treatment retention/discontinuation studies and calibrated to match observed treatment coverage. Represents a key target for interventions aimed at improving treatment retention.
+   * - ts
+     - T
+     - S
+     - Derived from DisMod-AT/NumPyro model and treatment outcome studies
+     - Treatment-associated recovery rate. Estimated from literature on MOUD outcomes and long-term recovery rates. Represents transition to sustained recovery while on treatment.
```

Comment:
There's an odd kind of fortune-telling going on here, because in reality you don't _know_ whether recovery is sustained until after it has been going on for a while, whereas here you know immediately?

Reply:
I think I'm going to make a structural change with assumptions that get around this... stay tuned